docs/sources/setup-grafana/configure-security/configure-authentication/grafana-com/index.md

```diff
@@ -32,17 +32,15 @@ client_secret = eyJ0Ijoib2F1dGgyYyIhlmlkIjoiNzUwYmMzM2MxMGRjMjE6NDh3OWQiLCJ2Ijoi
 scopes = user:email
 allowed_organizations = sampleorganization
 enabled = true
-
-# prevents the sync of org roles from Grafana.com
-skip_org_role_sync = false
 ```
 
 ## Skip organization role sync
 
-To prevent the sync of org roles from Grafana.com, set `skip_org_role_sync` to `true`. This is useful if you want to use Grafana.com as an identity provider but want to manage the org roles in Grafana.
+To prevent the sync of org roles from Grafana.com, set `skip_org_role_sync` to `true`. This is useful if you want to use your Grafana instance to manage the organization roles for your users.
```

Comment:
```suggestion
To prevent the sync of org roles from Grafana.com, set `skip_org_role_sync` to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.
```